### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by zchen-qumulo**

### CODE REVIEW

File: generate_qumulo_cloudformation_template_test.py [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+            'pairing_ratio': 2,
+            'working_spec': {'VolumeSize': 1},
+        }
+        json_config = json.dumps(config, indent = 4)
+        with open(self.file_path, 'w+') as config_file:
+            config_file.write(json_config)
+
+        with self.assertRaisesRegex(NoBackingVolumesException, 'The backing volumes'):
+            generate_qcft(2, self.file_path, 'st1', 'ami-123')
+
```
Comment (cwboden) [inline generate_qumulo_cloudformation_template_test.py:293]: I think you'll want
```python
if __name__ == '__main__':
    unittest.main()
```
At the end

File: qumulo_secret_rotation_lambda.py [hunk 2]
```diff
@@ -33,7 +33,7 @@
 
 import boto3
 
-from qumulo.rest_client import RestClient
```
Comment (cwboden) [inline qumulo_secret_rotation_lambda.py:36]: Any motivation behind changing this? I thought the `from` syntax is preferred, but both make sense

**Review by cwboden — APPROVED** [review]
Looks good!

I had a few open questions, but I think the code is in a great state 😁

File: generate_qumulo_cloudformation_template_test.py [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+import unittest
+from unittest.mock import call, patch, MagicMock
+
+from typing import Any, Dict, Optional
+
+from troposphere import Template
+
+from generate_qumulo_cloudformation_template import *
+
+@patch('troposphere.ec2.EBSBlockDevice')
```
[the remark this answers is not in the document]
Comment (zchen-qumulo) [reply generate_qumulo_cloudformation_template_test.py:12]: It is needed on every test, I just didn't need it to return a value most of the time.

File: qumulo_secret_rotation_lambda.py (hunk 2 above)
Comment (zchen-qumulo) [reply qumulo_secret_rotation_lambda.py:36]: The patch mocking didn't work properly when the place it was used wasn't fully typed out (qumulo.rest_client.RestClient).

I wasn't using just RestClient, so I changed the import syntax.

**Pull request merged**